stubs/BaseYii.phpstub

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
```suggestion
<?php
/**
 * Copyright 2021 Practically.io All rights reserved
 *
 * Use of this source is governed by a BSD-style
 * licence that can be found in the LICENCE file or at
 * https://www.practically.io/copyright/
 */
declare(strict_types=1);
```

Can we get the file header in here, please?

https://github.com/Practically/PsalmPluginYii2/blob/1.x/stubs/ActiveQuery.phpstub#L1-L9